sbt-ash/src/main/scala/org/virtuslab/ash/AkkaSerializationHelperPlugin.scala

```diff
@@ -0,0 +1,65 @@
+package org.virtuslab.ash
+
+import sbt.Def
+import sbt.Keys._
+import sbt._
+
+object AkkaSerializationHelperPlugin extends AutoPlugin {
+
+  object autoImport extends AkkaSerializationHelperKeys {
+    val baseAshSettings: Seq[Def.Setting[_]] = AkkaSerializationHelperPlugin.baseAshSettings
+  }
+  import autoImport.{baseAshSettings => _, _}
+
+  override def projectSettings: Seq[Def.Setting[_]] = additionalSettings ++ ashSettings
+
+  override def globalSettings: Seq[Def.Setting[_]] = Nil
+
+  lazy val additionalSettings: Seq[Def.Setting[_]] = Seq(
+    libraryDependencies ++= Seq(
+        compilerPlugin(ashDumpEventSchemaCompilerPlugin.value),
+        compilerPlugin(ashRegistrationCheckerCompilerPlugin.value),
+        compilerPlugin(ashSerializabilityCheckerCompilerPlugin.value)),
+    scalacOptions ++= Seq(
+        s"-P:dump-event-schema-plugin:${target.value}",
```

Comment:
Am I right that this covers #79?

Reply:
No, we just hide it under sbt plugin.
But in some sense also yes.